recipes-kernel/kernel-tests/kernel-tests-files/kernel_dmesg_diff.py

```diff
@@ -284,6 +284,10 @@ def parse_args():
                              'database, so the --skip_upload flag is not needed. This flag is useful for debugging issues with this test.')
     parser.add_argument('--skip_upload', help='Skip upload of dmesg record to database. Useful when debugging.', action="store_true")
     parser.add_argument('--suppress_diff_output', help='Reduces test output noise by removing the dmesg log diff from the output. Useful when debugging.', action="store_true")
+    parser.add_argument('--basis_kernel_dmesg_log_db_date', metavar="<date>",
```

Comment:
Change this argument name from basis_kernel_dmesg_log_db_date to basis_log_db_date. This better matches current_log_db_date. (The only difference between these option names should be to replace "current" with "basis".)

Reply:
Fixed.